### PR TITLE
Update the collisions Readme with the new replicator details

### PR DESCRIPTION
**Pull request by tahaislam**
## What this pull request accomplishes:

- Updates the collisions README to match the new replicator DAG that copies collisions tables from `move_staging` to `collisions` 

Closes issue #933

### CODE REVIEW

**Comment by gabrielwol** [comment]
Need to update with some new DAG details from #902

**Comment by chmnata** [comment]
@gabrielwol Can you update the doc with the new DAG stuff?

**Comment by gabrielwol** [comment]
@chmnata , @mkewins 
Added new details from recent PRs:
- https://github.com/CityofToronto/bdit_data-sources/pull/902
- https://github.com/CityofToronto/bdit_data-sources/pull/928

File: collisions/Readme.md [hunk 1]
```diff
@@ -40,30 +41,62 @@ Even though `collisions_replicator.acc_safe_copy` should not be directly queried
...
+--the bot needs to own the downstream table in order to update the table comment
+ALTER TABLE collisions.a OWNER TO collisions_bot;
+```
 
-The `collisions_replicator.ACC` table is dropped (or deleted) and recreated based on the table in `flashcrow` every night. From there, the `bigdata` postgres pipeline operates as follows:
-1. The table `collisions_replicator.acc_safe_copy` is updated based on any differences between it and `collisions_replicator.ACC` (this is known as an "upsert" query).
-2. The `collisions_replicator.collision_no` materialized view, which creates a unique id (UID) for each collision, is refreshed.
-3. The `collisions_replicator.events` and `collisions_replicator.involved` materialized views are refreshed (more on this below).
-4. New records, deletions and updates to existing records are tracked in a table called `collisions_replicator.logged_actions`.
+### Updating Existing Tables
```
Comment (chmnata) [inline collisions/Readme.md:101]: nice mention of the dependencies function :P

**Review by mkewins — APPROVED** [review]

**Comment by mkewins** [comment]
@gabrielwol I made a few small edits and changed some links. LGTM. Please merge + close as appropriate!

**Comment by gabrielwol** [comment]
Thanks for the edits Maddy!

**Pull request merged**